tableau_utilities/scripts/gen_config.py

```diff
@@ -21,6 +20,7 @@ def load_csv_with_definitions(file=None):
 
     """
 
+    definitions_mapping = dict()
```

Comment:
@JustinGrilli  This code isn't working anymore.  
* It's only making an empty dictionary for the column config
* It's not getting descriptions or folders out of an existing config that has both

Can you either fix this or just revert these changes?

Reply:
@jaybythebay Just pushed a fix for this!
I tested it on one of our datasources and it seems to be working properly now, but would be great to confirm if it is fixed for you as well :)